src/components/LeftSidebar/ConversationsListVirtual.vue

```diff
@@ -151,6 +163,14 @@ export default {
 				this.scrollToItem(index)
 			}
 		},
+
+		adjustScrollerHeight() {
+			const scrollerContainer = this.$refs.scroller.$el.querySelector(
+				'.vue-recycle-scroller__item-wrapper'
+			)
+			const minHeight = this.conversations.length * this.CONVERSATION_ITEM_SIZE + 2
+			scrollerContainer.style.minHeight = minHeight + 'px'
+		},
```

Comment:
We should not do this to fix the border...

1. It updates only on conversation list update, but it doesn't work on height change, e.g. window resize. It can be fixed by `window`'s `resize` event or resize observer, but it's not great for performance to do it just for a border.
2. It directly mutates DOM nodes that are rendered and managed by Vue with Virtual DOM. We should never mutate Vue's DOM. It has many ways to break.
3. It also searches for an element with a query selector, which is not a big issue but can be easily broken by a small layout change with a class rename.

Reply:
Ferdinand just suggested an alternative solution for such issues in `NcCheckboxRadioSwitch`: using [`outline-offset`](https://developer.mozilla.org/en-US/docs/Web/CSS/outline-offset) `-2px`. Then the outline it inside the element, and there is no issue with overflow. Maybe try using it here? Or fix it on the nextcloud/vue side...

Reply:
Or to try to add a 2px footer in the virtual scroller

Reply:
The `outline-offset` is the optimal solution IMO but would rather change it in nextcloud/vue . If so, I will also add the `overflow: hidden` there too.